extension/mysql_to_doris/user_define_tables.sh

```diff
@@ -0,0 +1,144 @@
+sed -i 's/\<blob\>/varchar(65533)/g' ./user_files/tables.sql
+sed -i 's/\<mediumblob\>/varchar(65533)/g' ./user_files/tables.sql
+sed -i 's/\<longblob\>/varchar(65533)/g' ./user_files/tables.sql
+sed -i 's/\<tinystring\>/varchar(65533)/g' ./user_files/tables.sql
+sed -i 's/\<mediumstring\>/varchar(65533)/g' ./user_files/tables.sql
+sed -i 's/\<longstring\>/varchar(65533)/g' ./user_files/tables.sql
+sed -i 's/\<timestamp\>/datetime/g' ./user_files/tables.sql
+sed -i 's/AUTO_INCREMENT//g' ./user_files/tables.sql
+sed -i 's/\<unsigned\>//g' ./user_files/tables.sql
+sed -i 's/\<zerofill\>//g' ./user_files/tables.sql
+sed -i 's/\<json\>/varchar(65533)/g' ./user_files/tables.sql
+sed -i 's/enum([^)]*)/varchar(65533)/g' ./user_files/tables.sql
+sed -i 's/\<set\>/varchar(65533)/g' ./user_files/tables.sql
+sed -i 's/\<bit\>/varchar(65533)/g' ./user_files/tables.sql
+sed -i 's/CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci//g'  ./user_files/tables.sql
+sed -i 's/CHARACTER SET utf8mb4 COLLATE utf8_general_ci//g' ./user_files/tables.sql
+sed -i 's/CHARACTER SET utf8 COLLATE utf8_general_ci//g' ./user_files/tables.sql
+sed -i 's/COLLATE utf8mb4_general_ci//g' ./user_files/tables.sql
+sed -i 's/COLLATE utf8_general_ci//g'  ./user_files/tables.sql
+sed -i 's/DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP//g' ./user_files/tables.sql
+sed -i 's/DEFAULT NULL ON UPDATE CURRENT_TIMESTAMP//g' ./user_files/tables.sql
+sed -i 's/CHARACTER SET utf8 COLLATE utf8_bin//g' ./user_files/tables.sql
+sed -i 's/COLLATE utf8_general_ci//g'  ./user_files/tables.sql
+sed -i 's/datetime([0-9])/varchar(65533)/g' ./user_files/tables.sql
+sed -i 's/CHARACTER SET utf8mb4 COLLATE utf8mb4_0900_ai_ci//g' ./user_files/tables.sql
+sed -i 's/\<binary\>/varchar(65533)/g' ./user_files/tables.sql
+sed -i 's/\<varbinary\>/varchar(65533)/g' ./user_files/tables.sql
+sed -i 's/binary([0-9])/varchar(65533)/g' ./user_files/tables.sql
+sed -i 's/varbinary([0-9])/varchar(65533)/g' ./user_files/tables.sql
+sed -i 's/string([0-9])/varchar(65533)/g' ./user_files/tables.sql
+sed -i 's/\<string\>/varchar(65533)/g' ./user_files/tables.sql
+sed -i 's/string([0-9])/varchar(65533)/g' ./user_files/tables.sql
+sed -i 's/binary([0-9][0-9])/varchar(65533)/g' ./user_files/tables.sql
+sed -i 's/varbinary([0-9][0-9])/varchar(65533)/g' ./user_files/tables.sql
+sed -i 's/string([0-9][0-9])/varchar(65533)/g' ./user_files/tables.sql
+sed -i 's/binary([0-9][0-9][0-9])/varchar(65533)/g' ./user_files/tables.sql
+sed -i 's/varbinary([0-9][0-9][0-9])/varchar(65533)/g' ./user_files/tables.sql
+sed -i 's/string([0-9][0-9][0-9])/varchar(65533)/g' ./user_files/tables.sql
+sed -i 's/CHARACTER SET utf8mb4 COLLATE utf8mb4_bin//g' ./user_files/tables.sql
+
+
+#######################################
+#import doris
+for table in $(awk -F '\n' '{print $1}' ./conf/tables)
+        do
+        echo "use $d_doris; drop table if exists ${table};" |mysql -h$master_host -P$master_port -uroot -p$doris_password 2>/dev/null
```

Comment:
for loop not ending

Reply:
the for loop read result set is limited，so it is auto over